felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/Key.kt

```diff
@@ -9,6 +9,7 @@ enum class Key(val str: String) {
     EVENT_NAME("@event_name"),
     BEHOV("@behov"),
     LØSNING("@løsning"),
+    EXTRA("@extra"),
```

Comment:
Er "extra" en egendefinert nøkkel? For da synes jeg at vi ikke burde ha "@".

Reply:
Jeg satt opp @ fordi jeg anser feltet som en systemverdi - noe som gjelder på tvers av alle meldinger. Hva er kriteriet for å bruke @ tenker du?

Reply:
Fjerner all bruk av @ i våre egne felter.